packages/client/components/ui/ToggleHomepage.tsx

```diff
@@ -0,0 +1,24 @@
+import { motion } from 'framer-motion';
+import { useState } from 'react';
+
+export default function ToggleHomepage() {
```

Comment:
@Artentii we prefer to use the arrow function, I have changed it to make all the code the same, example:

```js
const Test = () => {}
```

Reply:
Okey, I'll start using the arrow function then.